### PR TITLE
boleto: add fine and interest

### PR DESCRIPTION
**Pull request by jeanaugustoos**
## Description

Create fine and interest interface and send it to bradesco.

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/resources/boleto/model.js [hunk 1]
```diff
@@ -185,6 +187,14 @@ function create (database) {
       defaultValue: 0,
     },
 
+    interest: {
+      type: JSON,
```
Comment (lucianopf) [inline src/resources/boleto/model.js]: Não seria interessante colocar tbm um `defaultValue: {}`? 🤔

File: src/resources/boleto/model.js [hunk 2]
```diff
@@ -185,6 +187,14 @@ function create (database) {
...
+    },
+
+    fine: {
+      type: JSON,
```
Comment (lucianopf) [inline src/resources/boleto/model.js]: Não seria interessante colocar tbm um `defaultValue: {}`? 🤔

File: src/providers/bradesco/index.js [hunk 3]
```diff
@@ -46,6 +48,12 @@ const isLengthEquals = numberOfCharacters => pipe(
   equals(numberOfCharacters)
 )
 
+const normalizePercentage = pipe(
```
Comment (grvcoelho) [inline src/providers/bradesco/index.js]: Será que rola fazer um teste unitário dessa função, para garantir que ela cobre edge cases?

Comment (jeanaugustoos) [reply src/providers/bradesco/index.js]: Opa, feito :grimacing:

File: src/providers/bradesco/index.js [hunk 4]
```diff
@@ -41,11 +46,29 @@ const buildHeaders = () => {
...
+
 const isLengthEquals = numberOfCharacters => pipe(
   length,
   equals(numberOfCharacters)
 )
 
+const normalizePercentage = ifElse(
+  isNumber,
+  pipe(
+    percentage => Number(percentage).toFixed(5),
```
Comment (lucianopf) [inline src/providers/bradesco/index.js]: Boooa mano! 
Sério esse constructor de `Number` é muito mágico! 😱 
Tava crente que ele iria retornar um int e portanto não teria `.toFixed` mas aparentemente tem! 😬

File: src/providers/bradesco/index.js [hunk 5]
```diff
@@ -41,11 +46,29 @@ const buildHeaders = () => {
   }
 }
 
+const isNotNil = complement(isNil)
+
+const isNumber = both(
+  isNotNil,
+  complement(isNaN) // eslint-disable-line no-restricted-globals
```
Comment (lucianopf) [inline src/providers/bradesco/index.js]: Liiiinda mano, parabéns por tratar isso!

**Pull request merged**